bhamcal/extractor.py

```diff
@@ -69,3 +69,20 @@ def extract_datetime(date, time):
     dt = DEFAULT_TIMEZONE.localize(dt)
     dt = dt.astimezone(pytz.utc)
     return dt
+
+
+def clean_subject(subject: str) -> str:
+    # Was not needed in my case but I'll assume others do.
+    one = r"^LI "
+    # Remove module codes, LM and/or LH and extended.
+    two = r"(?P<code>\([0-9]+/[0-9]+\))|(?P<prefix>LM/LH|LH/LM|LH|LM)|(?P<extended>\(Extended\)?)"
```

Comment:
LI is another prefix, like LH or LM. So they should probably be handled similarly?